maven-modules/maven-exec-plugin/pom.xml

```diff
@@ -84,6 +84,7 @@
     <properties>
         <maven-compiler-plugin.version>3.12.1</maven-compiler-plugin.version>
         <exec-maven-plugin.version>3.1.0</exec-maven-plugin.version>
+        <junit-jupiter.version>5.10.0</junit-jupiter.version>
```

Comment:
Is it necessary to use 5.10.0 here? is 5.10.2 not working?

Reply:
I can't remember if I have asked about the same and you have already answered before though.